lighty-core/lighty-controller/pom.xml

```diff
@@ -47,6 +56,26 @@
             <groupId>org.opendaylight.infrautils</groupId>
             <artifactId>diagstatus-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.opendaylight.infrautils</groupId>
+            <artifactId>jobcoordinator-impl</artifactId>
+            <version>${infrautils-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.opendaylight.infrautils</groupId>
+            <artifactId>caches-guava-impl</artifactId>
+            <version>${infrautils-version}</version>
```

Comment:
remove all <version> declarations here -- version declarations are already correctly set (builds for me)